### PR TITLE
updating tox.ini for zuul

### PR DESCRIPTION
**Pull request by radez**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -14,11 +14,13 @@ setenv =
 deps = -r{toxinidir}/test-requirements.txt
 commands = python -m unittest discover
 
+[testenv:linters]
+basepython = python3
+commands = flake8 {posargs}
+
 [testenv:pep8]
```
Comment (pabelanger) [inline tox.ini]: I'd remove this entry point, and update the env_list on the top live to be linters, not pep8

Comment (radez) [reply tox.ini]: done, thx

**Comment by radez** [comment]
recheck

**Comment by radez** [comment]
recheck

**Pull request merged**